cluster/vm.json

```diff
@@ -10,6 +10,28 @@
     },
     "domain": {
       "devices": {
+        "disks": [
+          {
+            "device": "disk",
+            "driver": {
+              "name": "qemu",
+              "type": "raw"
+            },
+            "snapshot": "external",
+            "source": {
+              "host": {
+                "name": "iscsi-demo-target",
+                "port": "3260"
+              },
+              "name": "iqn.2017-01.io.kubevirt:sn.42/2",
+              "protocol": "iscsi"
+            },
+            "target": {
+              "dev": "vda"
```

Comment:
In the vagrant setup it works for me with vda/virtio,  hda/ide but not with sda/scsi.